src/renderToStream.ts

```diff
@@ -2,26 +2,28 @@ export { renderToStream }
 export { disable }
 
 import React from 'react'
-import { renderToPipeableStream, renderToReadableStream, version as reactDomVersion } from 'react-dom/server'
+import ReactDOMServer, { version as reactDomVersion } from 'react-dom/server'
```

Comment:
How about `import type { renderToPipeableStream as RenderToPipeableStream } from 'react-dom/server'` instead of `type RenderToPipeableStream = typeof import('react-dom/server').renderToPipeableStream`?